modules/claims_api/app/models/claims_api/form_526.rb

```diff
@@ -0,0 +1,77 @@
+
+# frozen_string_literal: true
+
+module ClaimsApi
+  class Form526
+    include ActiveModel::Validations
+    include ActiveModel::Conversion
+    extend ActiveModel::Naming
+
+    REQUIRED_FIELDS = %i[
+      veteran
+      applicationExpirationDate
+      serviceInformation
+      disabilities
+    ].freeze
+
+    BOOLEAN_REQUIRED_FIELDS = %i[
+      claimantCertification
+      standardClaim
+    ].freeze
+
+    NOT_REQUIRED_FIELDS = %i[
+      directDeposit
+      servicePay
+      treatments
+    ].freeze
+
+    (REQUIRED_FIELDS + BOOLEAN_REQUIRED_FIELDS + NOT_REQUIRED_FIELDS).each do |field|
+      attr_accessor field.to_sym
+    end
```

Comment:
This syntax also works, avoiding the manual iteration:

```
attr_accessor *REQUIRED_FIELDS, *BOOLEAN_REQUIRED_FIELDS, *NOT_REQUIRED_FIELDS
```

Maybe consider `OPTIONAL_FIELDS` instead of `NOT_REQUIRED_FIELDS` to avoid someone reading a usage of that name as a negation of the `REQUIRED_FIELDS` rather than a supplementary list.

Reply:
Done, thanks for the recommendations.

Reply:
Unfortunately rubocop doesn't like the creativity in *CONSTANTS so had to revert back to how I had it before.